src/css/variables.less

```diff
@@ -42,6 +46,12 @@
 @task-health-over-capacity-color: #4CA0FF;
 @task-health-unscheduled-color: #2A2C31;
 @task-health-running-color: #8E929B;
+@modal-dialog-bg-color: #FFFFFF;
```

Comment:
We have 6 different occurrences of `#FFFFFF` as a variable. Out-of-scope for this PR for sure, but I think it's about time we do an audit of our LESS variables and try to minimise variable proliferation. What do you think? @orlandohohmeier @aldipower